main/ui/src/main/java/org/cryptomator/ui/ExitUtil.java

```diff
@@ -202,4 +196,23 @@ private void restoreFromTray(ActionEvent event) {
 		});
 	}
 
+	public void updateTrayIcon(boolean areAllVaultsLocked) {
+		if (trayIcon != null) {
+			Image image = getAppropriateTrayIconImage(areAllVaultsLocked);
+			trayIcon.setImage(image);
+		}
+	}
+
+	private Image getAppropriateTrayIconImage(boolean areAllVaultsLocked) {
```

Comment:
Please use tabs instead of spaces (reminds me that we should commit the intellij code style settings with the project^^)

Reply:
Done